conduct.md

```diff
@@ -4,66 +4,85 @@ layout: page
 
 ## Code of Conduct
 
-DjangoCon Europe aims to be a welcoming event where people meet in a friendly environment.
-Accordingly, all participants are expected to show respect and courtesy to other participants
-throughout the conference.
+All delegates/attendees, speakers, exhibitors, organizers and
+volunteers at DjangoCon Europe 2018 are required to conform to the following Code of Conduct. Organizers will enforce this code before and throughout the event (including related activities such as the social events).
 
-To make clear what is expected, all delegates/attendees, speakers, exhibitors, organizers and
-volunteers at DjangoCon Europe 2018 event are required to conform to the following Code of Conduct.
-Organizers will enforce this code before and throughout the event (including related activites such
-as the social events).
+## Why do we have a Code of Conduct?
 
-### In short
+Our goals with having this CoC are:
 
-* DjangoCon Europe is dedicated to providing a harassment-free conference experience for everyone,
-  regardless of gender, sexual orientation, disability, physical appearance, body size, race, or
-  religion.
-* We do not tolerate harassment of conference participants in any form.
-* Sexual language and imagery are not appropriate for any conference venue, including talks, evening
-  events, and the conference hotel.
-* Be kind to others. Do not insult or put down other attendees. Behave professionally. Don’t publish
-  individual photographs of people without their consent. Remember that harassment and sexist,
-  racist, or exclusionary jokes are not appropriate for DjangoCon Europe.
-* Conference participants violating these rules may be sanctioned or expelled from the conference
-  without a refund at the discretion of the conference organizers.
+* Helping everyone feel safe and included. Many attendees are first timers, or have had poor experiences in other events. We want to set the expectation that harassment and other unpleasant behaviour are not acceptable. So that if people do have an unpleasant experience, they know that’s not the norm.
+* Helping to build trust that if an incident is reported, we will not respond with victim blaming, and that we will do a serious investigation. Even, for example, if the incident concerns someone in a position of power.
+* Informing everyone of the expected behaviour. We are a diverse community, and so it’s valuable to have the CoC as a basis for the expectations from everyone in our community.
+* Having a framework for report handling. The CoC is the first starting point for dealing with a report, assessing whether the CoC was violated, and what action should be taken.
```

Comment:
"Providing a framework for handling reports. The Code of Conduct contains not only expected behaviour, but also how we deal with reports, assessing whether the Code of Conduct was violated, and what actions will be taken.", maybe?

Reply:
I reworded this slightly, but did not add "how we deal with reports" or "what actions will be taken". The former belongs in a separate guide, to be written, and for the latter I intentionally included a non-limitative list later on, as we may need other actions.